benchmarks/DeepLearning/README.md

```diff
@@ -1,5 +1,12 @@
 # Deep Learning Benchmark
 
+## Model Level Benchmark
```

Comment:
Add the statement for Python (using model-level benchmark needs the Python environment), e.g., enter the Python environment, configure the Python path, etc.